portfolio/src/main/webapp/index.html

```diff
@@ -3,71 +3,88 @@
   <head>
     <meta charset="UTF-8">
     <title>Pablo's Portfolio</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css" integrity="sha384-0evHe/X+R7YkIZDRvuzKMRqM+OrBnVFBL6DOitfPri4tjfHxaWutUpFmBp4vmVor" crossorigin="anonymous">
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body id="background"> 
-    <div id="content">
-      <h1 id="header"><b>Pablo Dueñas' Portfolio:</b></h1>
-      <img id="mainImage" src="pablo-photos.JPG" alt="Photos of Pablo">
+  <body class="background"> 
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.bundle.min.js" integrity="sha384-pprn3073KE6tl6bjs2QrFaJGz5/SUsLqktiwsUTF55Jfv3qYSDhgCecCxMW52nD2" crossorigin="anonymous"></script>
```

Comment:
Nit: I don't think we need the <script> tag to re-download bootstrap since that's already done in line 6.